addons/pfconfig/comparator/dumper.pl

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+dumper.pl
+
+=head1 SYNOPSIS
+
+dumper.pl <codebase>
+
+=head1 DESCRIPTION
+
+Dumps the configuration for a codebase
+
+No need to use it directly, use addons/pfconfig/comparator/config-comparator.sh
+
+=cut
+
+use strict;
+use warnings;
+
+use Sereal::Encoder;
+use Data::Dumper;
+
+my $BASE = $ARGV[1];
+use lib $ARGV[0];
+
+my $ENCODER = Sereal::Encoder->new;
+our %configs;
+
+sub {
+  use pf::config;
+  my @exported = @pf::config::EXPORT;
+  my @badvalues = ('%ConfigProvisioning');
+  @exported = grep { !($_ ~~ @badvalues ) } @exported;
+  dump_module('pf::config', @exported);
+}->();
```

Comment:
I wanted to declare my variables each time and used it as a clear way to split the dumping in case we want to disable it. And scoping them also removes the dependencies of variables between sections since they're redeclared everytime. I may be doing a bit too much javascript so that's why it seems normal to my eyes. I can remove these if it really bothers everyone.